desktop/cmp/treemap/TreeMapModel.js

```diff
@@ -370,14 +372,15 @@ export class TreeMapModel extends HoistModel {
         // 2) Transform heatValue into a normalized colorValue, according to the colorMode.
         const maxHeat = isFinite(this.maxHeat) ? this.maxHeat : max(heatValues);
         data.forEach(it => {
-            const {heatValue} = it;
+            const {heatValue, record} = it,
+                value = record.data[valueField];
 
-            if (!this.valueIsValid(heatValue)) {
+            if (!this.valueIsValid(heatValue) || (heatValue === 0 && value === 0)) {
```

Comment:
I'm not sure how we want to handle situations where both the heatValue and the value at the valueField equal 0, especially for split tree maps.